src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -34,8 +34,9 @@
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
                         <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                                    Name="ContextMenuPanel">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding ElementName=ContextMenuPanel, Path=ActualWidth}" />
```

Comment:
Thanks a lot for trying this out, @aosyatnik. I was just trying to get rid of the binding expression if it is not necessary, but since we can't do that, I'm okay with the binding for now.